src/engines/kotor/gui/chargen/chargeninfo.h

```diff
@@ -38,6 +40,9 @@ class Creature;
 
 class CharacterGenerationInfo {
 public:
+	CharacterGenerationInfo(const CharacterGenerationInfo &);
+	void operator=(const CharacterGenerationInfo &);
```

Comment:
Don't leave out the parameter names here. I.e. it should be `CharacterGenerationInfo(const CharacterGenerationInfo &info);` and `void operator=(const CharacterGenerationInfo &info);`